src/opa/rego/policy.json

```diff
@@ -0,0 +1,7 @@
+{
+  "authorization_rules": [    
+    {"route": "/users", "roles": ["admin"]},
+    {"route": "/order", "roles": ["contributor"]},
+    {"route": "/users/*/friends", "roles": ["admin", "contributor"]}

```

Comment:
To get this working end-to-end we'll need to add some of the routes in the conference API

Reply:
I changed routes in policy.json using conference API's routes